erizo/src/erizo/Stats.cpp

```diff
@@ -26,15 +25,29 @@ namespace erizo {
   }
 
   uint32_t Stats::processRtpPacket(char* buf, int len) {
-    rtpBytesReceived_+=len;
+    RtcpHeader *chead = reinterpret_cast<RtcpHeader*> (buf);
```

Comment:
I think we need to protect some of this code with a mutex. And also check that we're protecting getStats.